XamlStyler.Mac/Gui/XamlStylerOptionsWidget.cs

```diff
@@ -0,0 +1,200 @@
+﻿using Gdk;
+using System;
+using System.ComponentModel;
+using System.Linq;
+using Xavalon.XamlStyler.Mac.Utils;
+using Gtk;
+using Xavalon.XamlStyler.Mac.ViewModels;
+
+namespace Xavalon.XamlStyler.Mac.Gui
+{
+	[ToolboxItem(true)]
+	public partial class XamlStylerOptionsWidget : Bin
+	{
+		private readonly XamlStylerOptionsViewModel _viewModel;
+		private readonly Color _groupHeaderColor;
+		private readonly Color _altRowColor;
+
+		public XamlStylerOptionsWidget(XamlStylerOptionsViewModel viewModel)
+		{
+			_viewModel = viewModel;
+
+            if (MonoDevelop.Ide.IdeApp.Preferences.UserInterfaceTheme == MonoDevelop.Ide.Theme.Dark)
+            {
+                _groupHeaderColor = ColorUtils.Parse("#222222");
```

Comment:
can these hard-coded values be stored in variables somewhere?

Reply:
moved it to separate Colors class